src/components/Header.tsx

```diff
@@ -0,0 +1,63 @@
+import React, {
+  FC,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  addTodo: (todo: Todo) => void;
+};
+
+export const Header: FC<Props> = ({ addTodo }) => {
+  const [inputValue, setInputValue] = useState('');
+  const newTodoField = useRef<HTMLInputElement>(null);
+
+  const handleOnChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = event.target;
+
+    setInputValue(value);
+  };
+
+  const onFormSubmit = (event: React.FormEvent<HTMLFormElement>) => {
```

Comment:
```suggestion
  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
    const { value } = event.target;

    setInputValue(value);
  };

  const handleFormSubmit = (event: React.FormEvent<HTMLFormElement>) => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364

Reply:
fixed